src/indicators/singleValuedIndicator.js

```diff
@@ -0,0 +1,75 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.indicators.singleValuedIndicator = function(algorithm) {
```

Comment:
I can see why the algorithm is passed in the constructor (it is required to function) but other components have similar restrictions and just default it to something sensible (e.g. series default x/y scales). Consider removing it?